### DISCUSSION TITLE
Refund please! the application is not working as expected.

### DISCUSSION THREAD

**Discussion by CevaGreu** [aclap-dev/video-downloadhelper#910] [Help and Q&A]
### Your question

I bought it today to download some videos, but it's not working at all. I have the pop-up to download the videos, but after downloading, the video contains just a grey image with no sound.
How can i get the refund?

### Addon & CoApp version and details

version: 9.0.2.3
target: google
channel: stable
lang: en-US
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"bd42e475df994ceb"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": true,
    "open_count_store": 14,
    "successfull_dl": 48
}

**Comment by jcfrog** [comment]
hi,
please send an email at licenses@downloadhelper.net
